cmd/util_test.go

```diff
@@ -55,3 +55,40 @@ func (s *utilsSuite) TestProxyFields(c *check.C) {
 		}
 	}
 }
+
+func (s *utilsSuite) TestVerifyPdEndpoint(c *check.C) {
+	defer testleak.AfterTest(c)()
+	// empty URL.
+	url := ""
+	c.Assert(verifyPdEndpoint(url, false), check.ErrorMatches, ".*PD endpoint should be a valid http or https URL.*")
+
+	// invalid URL.
+	url = "\n hi"
```

Comment:
```suggestion
	url = "\r hi"
```
ref: https://github.com/pingcap/ticdc/pull/2184/commits/e45f66a610c8d8a74620ab333cea7536612fe22c